compiler/bcq-tools/README.md

```diff
@@ -76,3 +76,38 @@ const(value=[1, 2, 3, -3], name='const3')
 ### Caution
 
 - Generated output_arrays will be start with comma.
+
+## generate_bcq_metadata
+
+### Purpose
+
+To apply BCQ, additional information which is not included in original model is needed.
+For example, without metadata, we cannot know from where to where BCQ related nodes are included. Metadata for BCQ is needed due to those reason and `generate_bcq_metadata` will generate metadata in the model according to each BCQ version.
+
+### How to use
+
+```bash
+generate_bcq_metadata \
+--input_path /path/to/original_model.pb \
+--output_path /path/to/metadata_inserted_model.pb
+--output_arrays output1,output2,...,outputN
+```
+
+### How it works
+
+First of all, metadata will be generated as following description.
+```
+< Generated Metadata in BCQ version 1 >
+[0] Starting magic number                = {-2e9 + 27}
+[1] Version of BCQ                       = {1}
+[2] The number of original model outputs = {N | N > 0}
+[3] Bundle size                          = {7, 8}
+[4] Ending magic number                  = {2e9 - 27}
+```
+- BCQ version 1
+    - Two magic numbers, starting and ending magic number, are used for indicating that the node includes BCQ metadata. To decrease value duplication probability, prime number is used and the value is inserted not only at the beginning but also at the end.
```

Comment:
`the node includes` : I'm don't quite understand what the `node` is.

Reply:
I will change to `the model includes`